docs/releases.rst

```diff
@@ -3,6 +3,56 @@
 Release notes
 =============
 
+Version 3.0.0
+-------------
+
+This version adds integration with `Citus <https://www.citusdata.com>`__ and makes it possible to survive temporary DCS outages without demoting primary.
+
+.. warning::
+   - Version 3.0.0 is the last release supporting Python 2.7. Upcoming release will drop support of Python versions older than 3.7.
+
+   - The RAFT support is deprecated. We will do our best maintaining it, but take neither guaranty nor responsibility on possible issues.
+
+   - This version is the first step in getting rid of the "master", in favor of "primary". Upgrading to the next major release will work reliably only if you run at least 3.0.0.
+
+
+**New features**
+
+- DCS failsafe mode (Alexander Kukushkin, Polina Bungina)
+
+  If the feature is enabled it will allow Patroni cluster survive temporary DCS outages. You can find more details in the :ref:`documentation <dcs_failsafe_mode>`.
```

Comment:
```suggestion
  If the feature is enabled it will allow Patroni cluster to survive temporary DCS outages. You can find more details in the :ref:`documentation <dcs_failsafe_mode>`.
```